sys/include/net/netstats.h

```diff
@@ -18,13 +18,22 @@
  * @author      Oliver Hahm <oliver.hahm@inria.fr>
  */
 
+#include <stdint.h>
+
 #ifndef NETSTATS_H
 #define NETSTATS_H
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+typedef enum {
+    NETSTATS_LAYER2     = 0x01,
+    NETSTATS_IPV6       = 0x02,
+    NETSTATS_ALL        = 0xFF
```

Comment:
Hm, IIRC the reason for choosing `0xFF` here was to enable checks like `if (module & NETSTATS_IPV6)` to return true for `NETSTATS_ALL`.